collector/config.yml

```diff
@@ -25,7 +25,7 @@ meta_servers:
   - 127.0.0.1:34603
 
 # local server port
-port : 34101
+port : 8081
```

Comment:
8080, 8081 are widely used ports for web servers, it would be better to use 34101 to keep compatible (old version of collectors may have been used by communituy users, don't change the behavior).